features/order/managing_orders/seeing_order_with_items.feature

```diff
@@ -0,0 +1,26 @@
+@managing_orders
+Feature: Seeing an order with its items
+    In order to see ordered products
+    As an Administrator
+    I want to be able to list items
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "Angel T-Shirt" priced at "€39.00"
+        And the store has a product "Angel Mug" priced at "€19.00"
+        And the store ships everywhere for free
+        And the store allows paying with "Cash on Delivery"
+        And there is a customer "lucy@teamlucifer.com" that placed an order "#00000666"
+        And the customer chose "Free" shipping method to "United States" with "Cash on Delivery" payment
+        And the customer bought a single "Angel T-Shirt"
+        And the customer bought a single "Angel Mug"
```

Comment:
Anyway, I know we use that phrase heavily, but IMO `the customer bought "Angel T-Shirt"` is more natural (then `the customer bought [0-9]+ :productName` for multiple units).